kimchi/src/circuits/constraints.rs

```diff
@@ -656,7 +656,7 @@ impl<F: PrimeField + SquareRootField> Builder<F> {
         //~ 2. Create a domain for the circuit. That is,
         //~    compute the smallest subgroup of the field that
         //~    has order greater or equal to `n + ZK_ROWS` elements.
-        let domain = EvaluationDomains::<F>::create(gates.len() + ZK_ROWS as usize)?;
+        let domain = EvaluationDomains::<F>::create(gates.len() + (ZK_ROWS) as usize)?;
```

Comment:
?

Reply:
number of such changes below as well

Reply:
Done https://github.com/o1-labs/proof-systems/pull/974/commits/e8717474ce86614b4db72b3e06dc19e9de9cd469